test/tests/serial/tests.bats

```diff
@@ -416,12 +414,10 @@ EOF
   sleep 10
 
   # send some big emails
-  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
-  assert_success
-  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
-  assert_success
-  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
-  assert_success
+  _send_emails 'email-templates/quota-exceeded' '0.0.0.0 25'
+  _send_emails 'email-templates/quota-exceeded' '0.0.0.0 25'
+  _send_emails 'email-templates/quota-exceeded' '0.0.0.0 25'
```

Comment:
I'm confused why this didn't cause tests to fail? `_send_emails` looks like a typo to me. Ohh.. it's because of the `skip` :man_facepalming: 

```suggestion
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
```

Reply:
Good catch though!